snapshots/health/2024-04-22/gpei_funding.xlsx.dvc

```diff
@@ -0,0 +1,27 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: GPEI Historical Contributions
+    date_published: "2022-12-31"
```

Comment:
Seems that the data was published in Oct or Nov 2023, because the [Tableau dashboard](https://public.tableau.com/app/profile/publish.gpei/viz/GPEIHistoricPlus_2022/GPEIHistoricalContributions) is from Oct and [the doc](https://polioeradication.org/wp-content/uploads/2023/11/GPEI-Historical-Contributions-v18-20221231.pdf) was published in Nov (by looking at the link)

Reply:
Good catch!